apps/node-red/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.7"
+
+services:
+  web:
+    image: nodered/node-red:2.0.6-12
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      # Replace <port> with the port that your app's web server
+      # is listening inside the Docker container. If you need to
+      # expose more ports, add them below.
```

Comment:
You can remove all commments out of this file, they aren't needed.

Reply:
Done. I've added some environment variables that could be useful in the future for node-red flows. I haven't added the APP_HIDDEN_SERVICE or the tor environment variables. Seems too risky to expose them. If users will have a specific scenario where they need them, we could add these variables in the future.